.github/workflows/tlaplus.yml

```diff
@@ -114,3 +115,129 @@ jobs:
           path: |
             tla/consistency/*_TTrace_*.tla
             tla/*.json
+
+  model-checking-consensus:
+    name: Model Checking - Consensus
+    runs-on: [self-hosted, 1ES.Pool=gha-virtual-ccf-sub]
+    container:
+      image: ghcr.io/microsoft/ccf/ci/default:build-26-06-2024
```

Comment:
Would be best to move to a workflow variable: https://docs.github.com/en/actions/learn-github-actions/variables#defining-environment-variables-for-a-single-workflow

That way updating the image is a one line change.

Reply:
It seems it doesn't work in
```container:
  image:
```
block, it's not visible in there.

Reply:
That is quite sad.